widget/menu.go

```diff
@@ -18,6 +18,7 @@ type Menu struct {
 	OnDismiss   func()
 	activeItem  *menuItem
 	customSized bool
+	hiding      bool
```

Comment:
I'm not sure how this is better than the bool in basewidget. Was locking the issue?

Reply:
That is a good question indeed. The internal bool was being used before and it didn't work. This line had to change to avoid it:

```go
	return i.Child() != nil && i.Child().Visible()
	return i.Child() != nil && !i.Child().hiding
```